lib/v2/logclub/report.js

```diff
@@ -0,0 +1,98 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const { id = 'Report' } = ctx.params;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 11;
+
+    const rootUrl = 'https://www.logclub.com';
+    const currentUrl = new URL('lc_report', rootUrl).href;
+    const apiUrl = new URL(`front/lc_report/load${id}List`, rootUrl).href;
+
+    const { data: response } = await got.post(apiUrl, {
+        json: {
+            page: 1,
+        },
+    });
+
+    let items = response.list.slice(0, limit).map((item) => ({
+        title: item.title,
+        link: new URL(`front/lc_report/get_report_info/${item.id}`, rootUrl).href,
+        description: art(path.join(__dirname, 'templates/description.art'), {
+            image: {
+                src: item.img_url?.split(/\?/)[0] ?? undefined,
+                alt: item.title,
+            },
+        }),
+        author: item.author,
+        category: [item.channel_name],
+        guid: `logclub-report-${item.id}`,
+        pubDate: timezone(parseDate(item.release_time), +8),
+    }));
+
+    items = await Promise.all(
+        items.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: detailResponse } = await got(item.link);
+
+                const content = cheerio.load(detailResponse);
+
+                content('p img').each((_, el) => {
+                    el = content(el);
+                    el.parent().replaceWith(
+                        art(path.join(__dirname, 'templates/description.art'), {
+                            image: {
+                                src: el.prop('src')?.split(/\?/)[0] ?? undefined,
+                                alt: el.prop('title'),
+                            },
+                        })
+                    );
+                });
```

Comment:
This remove rest of the images from first item (https://www.logclub.com/front/lc_report/get_report_info/3665) in `/logclub/lc_report`

Reply:
👌